python/tests/calculators.py

```diff
@@ -5,10 +5,35 @@
 from rascaline import SortedDistances
 from rascaline.calculator import DummyCalculator
 
+from rascaline.clib import _get_library, _set_default_logging_callback
+import ctypes
+
 from test_systems import TestSystem
 
 
+
 class TestDummyCalculator(unittest.TestCase):
+    def test_callback(self):
+        recorded_levels = []
+        recorded_messages = []
+        def test_callback(level, message):
+            recorded_levels.append(level)
+            recorded_messages.append(message)
```

Comment:
This might be easier to test by combining level & messages with something like
```suggestion
        recorded_log_events = []
        def test_callback(level, message):
            recorded_log_events.append((level, message))
```

And then check that we get `(log_level_info, targeted_message)` in the recorded events

Reply:
changed

Reply:
I don't see it?

Reply:
still open, forgot this

Reply:
done